workspaces/libnpmexec/lib/run-script.js

```diff
@@ -60,11 +61,13 @@ const run = async ({
       binPaths,
       event: 'npx',
       args,
-      stdio: 'inherit',
+      stdio,
       scriptShell,
     })
   } finally {
-    npmlog.enableProgress()
+    if (stdio === 'inherit') {
```

Comment:
this one needs to be removed also

Reply:
This isn't npm log output, this is progress bars specifically.

To suppress all logging we need `npmlog.pause` and `npmlog.resume`, but these should be independent of enabling/disabling progress.

Reply:
I don't want to install npmlog directly to pause it.
What about a flag to ignore npmlog?

Reply:
You already have it https://github.com/npm/cli/blob/e4cd2c6393827e2f5760aab731ed64cd2ab49c86/workspaces/libnpmexec/lib/run-script.js#L4

Reply:
I mean, I have ora as progress bar.
npmlog and ora are mangling the output.
I need to install and import npmlog in my project to pause the npmlog.
And that does not guarantee that the npmlog is muted due to hoisting.